dangerzone/isolation_provider/container.py

```diff
@@ -0,0 +1,317 @@
+import gzip
+import json
+import logging
+import os
+import pipes
+import platform
+import shutil
+import subprocess
+import tempfile
+from typing import Callable, List, Optional, Tuple
+
+import appdirs
+
+from ..document import Document
+from ..util import get_resource_path, get_subprocess_startupinfo
+from . import IsolationProvider
```

Comment:
Oh, there's one thing I realized:

It's weird practice to have a submodule do an import from `__init__.py`. In general, it's preferable to have a `base.py` module, and import that. The reason is that you may want to re-export the isolation provider classes in `__init__.py`, and doing so would lead to cyclic references.

Reply:
Just added that in [fd9bd37](https://github.com/freedomofpress/dangerzone/pull/302/commits/fd9bd37cb394b774da664c285cdd1c67d0018aa2).

Reply:
Niceeee... but! The CI tests are failing due to this move. Once you fix them, feel free to merge.